packages/cozy-stack-client/src/SharingCollection.js

```diff
@@ -142,7 +172,10 @@ SharingCollection.normalizeDoctype = DocumentCollection.normalizeDoctypeJsonApi
 
 // Rules determine the behavior of the sharing when changes are made to the shared document
 // See https://github.com/cozy/cozy-stack/blob/master/docs/sharing-design.md#description-of-a-sharing
-const getSharingRules = (document, sharingType) => {
+export const getSharingRules = (document, sharingType) => {
+  console.warn(
+    'sharingType is deprecated. We now set default policy. If we need to set a custom policy use the full `rules` object of the SharingCollection.create() method'
```

Comment:
Since the default policy is dependent on the document_type, I think it could be useful to output it here ?

`sharingType is deprecated. The default policy for ${document._type} is now ${policy}. To set a custom policy, use the `rules` object of the SharingCollection.create method.` 

I see below that one of the cases io.cozy.files/"items" does not use getSharingPolicy ? Is it wanted ?